modules/openapi-generator/src/main/resources/C-libcurl/CMakeLists.txt.mustache

```diff
@@ -92,8 +92,54 @@ target_link_libraries(${pkgName} ${CURL_LIBRARIES} )
 
 include(PostTarget.cmake OPTIONAL)
 
-#install library to destination
-install(TARGETS ${pkgName} DESTINATION ${CMAKE_INSTALL_PREFIX})
+if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/Config.cmake.in)
```

Comment:
How the `${CMAKE_CURRENT_SOURCE_DIR}/Config.cmake.in` is generated ?

In my env, this file doesn't exist, at this case, how to execute the original `install()` commands after your PR ?

Reply:
I think that I need one more day to take care of these points. Thank you for the patience.

Reply:
It should be created manually just like this https://github.com/kubernetes-client/c/blob/4221f70114de95773d3ef53e8cafe357f454b705/kubernetes/Config.cmake.in
But you are right, we should have a fallback when the file does not exist.

Reply:
Is the file `Config.cmake.in` is used for `find_package()` by users ?
https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html
Then the name should be `<PackageName>Config.cmake`, so I think the name `${pkgName}Config.cmake.in` is better than `Config.cmake.in` here.